Description: KWindowSystem doesn't probably work for us here, always use longer delay.
Author: Tuomas Nurmi <tuomas.nurmi@opinsys.fi>
Last-Update: 2022-11-14

--- a/src/SpectacleCore.cpp
+++ b/src/SpectacleCore.cpp
@@ -278,7 +278,7 @@
     // settings (and unless the user has set an extremely slow effect), 200
     // milliseconds is a good amount of wait time.

-    auto lMsec = KWindowSystem::compositingActive() ? 200 : 50;
+    auto lMsec = KWindowSystem::compositingActive() ? 500 : 500;
     QTimer::singleShot(theTimeout + lMsec, this, [this, lGrabMode, theIncludePointer, theIncludeDecorations]() {
         mPlatform->doGrab(Platform::ShutterMode::Immediate, lGrabMode, theIncludePointer, theIncludeDecorations);
     });
